CourseApp/GameCharacter.cs

```diff
@@ -0,0 +1,149 @@
+﻿namespace CourseApp
+{
+    using System;
+
+    public abstract class GameCharacter
+    {
+        private string skillone;
+        private string skilltwo;
+        private string skillthree;
+        private string ultimate;
+        private string weapon;
+        private int health;
+        private int damage;
+
+        public GameCharacter(int health, int damage)
+        {
+            Health = health;
+            Damage = damage;
+        }
+
+        public int Health
+        {
+            get
+            {
+                return this.health;
+            }
+
+            set
+            {
+                this.health = value;
+            }
+        }
+
+        public int Damage
+        {
+            get
+            {
+                return this.damage;
+            }
+
+            set
+            {
+                this.damage = value;
+            }
+        }
+
+        public string Skill_One
```

Comment:
А когда еще скиллы добавятся? 
```suggestion
        public string SkillOne
```